### DISCUSSION TITLE
Think about merging admin_audit log with nextcloud log

### DISCUSSION THREAD

**Discussion by szaimen** [nextcloud/all-in-one#2744] [Ideas] [answered]
See https://github.com/nextcloud/documentation/pull/10604

This makes the log visible in the logreader app.

However it does not show only warnings anymore in the logreader which might not be expected.

Better would be something like https://github.com/nextcloud/logreader/pull/857 but it does not seem to get any attention.

Need to think about this.

**Comment by szaimen** [comment]
@Zoey2936 what do you think about this idea?

**Reply by Zoey2936** [reply]
Not sure if that could be too much logging... the logreader PR seems to be test best way

**Reply by szaimen** [reply, marked as answer]
Yeah I think so as well. Lets not do this then for now.
